libsel4/arch_include/arm/interfaces/sel4arch.xml

```diff
@@ -15,7 +15,7 @@
             <description>
                 Takes a VSpace capability as an argument,
                 and installs a reference to the invoked
-                <texttt text='PageTable'/> in the VSpace according to the provided virtual address.
+                <texttt text='Page Table'/> in the VSpace according to the provided virtual address.
```

Comment:
Needs slightly more reformulation. What is being invoked is a PageTableCap, but the reference being installed is to a page table (the one the cap points to)